### PR TITLE
KakutesH7/H7v2/H7mini: use EKF2

### PR DESCRIPTION
**Pull request by julianoes**
This switches Kakute F7 and H7v1 to EKF2 to match Kakute H7v2 and mini.

Untested, could you test @vincentpoont2 ?

### CODE REVIEW

File: boards/holybro/kakutef7/init/rc.board_defaults [hunk 1]
```diff
@@ -12,12 +12,10 @@ param set-default CBRK_SUPPLY_CHK 894281
 # Select the Generic 250 Racer by default
 param set-default SYS_AUTOSTART 4050
 
-# use the Q attitude estimator, it works w/o mag or GPS.
-param set-default SYS_MC_EST_GROUP 3
-param set-default ATT_W_ACC 0.4
-param set-default ATT_W_GYRO_BIAS 0
-
+# use EKF2 without mag
```
Comment (bresch) [inline boards/holybro/kakutef7/init/rc.board_defaults:15]: you might also want to set `EKF2_MAG_TYPE 5` (none)

File: boards/holybro/kakutef7/default.px4board [hunk 2]
```diff
@@ -17,11 +17,11 @@ CONFIG_DRIVERS_OSD_ATXXXX=y
 CONFIG_DRIVERS_PWM_OUT=y
 CONFIG_DRIVERS_RC_INPUT=y
 CONFIG_DRIVERS_TELEMETRY_FRSKY_TELEMETRY=y
-CONFIG_MODULES_ATTITUDE_ESTIMATOR_Q=y
 CONFIG_MODULES_BATTERY_STATUS=y
 CONFIG_MODULES_COMMANDER=y
 CONFIG_MODULES_CONTROL_ALLOCATOR=y
 CONFIG_MODULES_DATAMAN=y
+CONFIG_MODULES_EKF2=y
```
Comment (bkueng) [inline boards/holybro/kakutef7/default.px4board:24]: Can you disable these ekf parts: https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v2/default.px4board#L26-L31?

File: boards/holybro/kakutef7/init/rc.board_defaults (hunk 1 above)
Comment (julianoes) [reply boards/holybro/kakutef7/init/rc.board_defaults:15]: I didn't do that for the other Kakutes either. So you're saying I should?

File: boards/holybro/kakutef7/default.px4board (hunk 2 above)
Comment (julianoes) [reply boards/holybro/kakutef7/default.px4board:24]: Ok, even if a GPS might be connected potentially? And would you do it for all Kakutes or just the F7?

Comment (bkueng) [reply boards/holybro/kakutef7/default.px4board:24]: GPS will still work. We can do it for all of them as they're generally low on flash.

File: boards/holybro/kakutef7/init/rc.board_defaults (hunk 1 above)
Comment (bresch) [reply boards/holybro/kakutef7/init/rc.board_defaults:15]: Nope, in fact it's fine, the EKF will still initialize the heading using the yaw estimator if there is no mag data: https://github.com/PX4/PX4-Autopilot/blob/d4c7ea74ef470930500c512d1b82bdc1c16752b0/src/modules/ekf2/EKF/gps_control.cpp#L126-L141

File: boards/holybro/kakutef7/default.px4board (hunk 2 above)
Comment (julianoes) [reply boards/holybro/kakutef7/default.px4board:24]: Alright, will do. Thanks.

Comment (julianoes) [reply boards/holybro/kakutef7/default.px4board:24]: @bkueng it looks like there is no way to fit EKF2 onto 1MB flash on the KakuteF7. Any ideas what to do? Retire it, or going back to Q?

Comment (bkueng) [reply boards/holybro/kakutef7/default.px4board:24]: Let's keep Q then.

**Comment by vincentpoont2** [comment]
@julianoes @bresch Please merge.
@julianoes Can we backport this to 1.14.1?

**Comment by bkueng** [comment]
@vincentpoont2 did you test this?

**Comment by vincentpoont2** [comment]
> @vincentpoont2 did you test this?

Yes, we did a test flight. The only thing we notice is the following warning.

**Comment by bkueng** [comment]
Thanks @vincentpoont2, then let's get this merged.
For reference, can you share the link to the log?

**Comment by vincentpoont2** [comment]
> Thanks @vincentpoont2, then let's get this merged.
> For reference, can you share the link to the log?

https://logs.px4.io/plot_app?log=82c6495e-4196-4985-aa9d-8b8a9ebfe7c6

**Pull request merged**